src/main/java/lotto/domain/Lotto.java

```diff
@@ -1,62 +1,68 @@
 package lotto.domain;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
 
 public class Lotto {
-    private static final int LOTTO_TICKET_PRICE = 1000;
-    private static final int LOTTO_BOUND = 45;
-    private static final int LOTTO_NUMBER_COUNT = 6;
-    private static Random random = new Random();
+    private static final int MAX_NUMBER = 45;
+    private static final int MAX_GENERATE_COUNT = 6;
 
-    private Lotto() {
-        throw new UnsupportedOperationException("해당 클래스는 인스턴스를 생성할 수 없습니다.");
+    private final Set<Integer> numbers;
```

Comment:
코멘트로도 의견 드렸는데 로또 번호 자체가 객체로 생성되면 어떨까 의견 드려봅니다 🙏 
그럼 조금 더 정리가 될 것 같아요 😄 
```suggestion
    private final Set<LottoNumber> numbers;
```